package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "azure-samples-for-js-management",
+  "version": "1.0.0",

```

Comment:
As the SDK is in preview, so for the samples, shall we also mark them as preview?  @nickzhums

Reply:
what is this name for?

Reply:
This is just the package name for this sample repo.